conductor-client/src/main/kotlin/com/openlattice/directory/Auth0UserDirectoryService.kt

```diff
@@ -47,7 +47,7 @@ class Auth0UserDirectoryService(
 
         var page = 0
         var pageOfUsers = auth0ManagementApi.searchAllUsers(searchQuery, page++,
-                                                            DEFAULT_PAGE_SIZE
+                                                            DEFAULT_PAGE_SIZE, "v3"
```

Comment:
It might be good to store like `AUTH0_API_VERSION` as a static constant in Auth0ManagementApi that's referenced here just so that piece is tightly coupled and to reduce the number of hardcoded strings